2-projects/1-todo-list/script.js

```diff
@@ -17,9 +58,21 @@ function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+  let todoInput = document.getElementById("todoInput");
+  let newTodo = input.value;
+  todos.push({ task: input.value, completed: false });
+  populateTodoList(todos);
+  todoInput.value = "";
 }
 
 // Advanced challenge: Write a fucntion that checks the todos in the todo list and deletes the completed ones (we can check which ones are completed by seeing if they have the line-through styling applied or not).
 function deleteAllCompletedTodos() {
   // Write your code here...
+  let li = document.querySelectorAll("li");
+  let list = document.getElementById("todo-list");
+  li.forEach((task) => {
+    if (task.style.textDecoration === "line-through") {
+      list.removeChild(task);
+    }
+  });
 }
```

Comment:
@qingwei91 
Could you please have a look at this exercise.
I'm not able to pass any new task to the list.
Thank you

Reply:
Thanks for checking.
It was an object which I intend to push new task to it.
I have fixed it thanks :)